templates/logrotate.j2

```diff
@@ -2,7 +2,7 @@
 # Slurm Logrotate Configuration
 ##
 # TODO: this ignores the actual *LogFile values
-{{ '/var/log/slurm-llnl' if __slurm_debian else '/var/log/slurm' }}/*.log {
+{{ '/var/log/slurm-llnl' if (not __slurm_configure_repos and __slurm_debian) else '/var/log/slurm' }}/*.log {
```

Comment:
Should it be `slurm_configure_repos` instead of `__slurm_configure_repos`?

Reply:
However, once again this is an opiniated change. It is legit to built our own package and use a different path than `/var/log/slurm`.

Reply:
Well, that should work in 99% of the cases, so let's keep this. We'll see later if upstream will comment on this one. However you may need to update the variable to remove the prefix `__`.